src/gridcoin/upgrade.cpp

```diff
@@ -163,26 +164,35 @@ bool Upgrade::CheckForLatestUpdate(std::string& client_message_out, Upgrade::Upg
         return false;
     }
 
-    if (!NewVersion) return false;
-
-    // New version was found
+    // Populate client_message_out regardless of whether new version is found, because we are using this method for
+    // the version information button in the "About Gridcoin" dialog.
     client_message_out = _("Local version: ") + strprintf("%d.%d.%d.%d", CLIENT_VERSION_MAJOR, CLIENT_VERSION_MINOR,
                                                           CLIENT_VERSION_REVISION, CLIENT_VERSION_BUILD) + "\r\n";
     client_message_out.append(_("GitHub version: ") + GithubReleaseData + "\r\n");
-    client_message_out.append(_("This update is ") + GithubReleaseType + "\r\n\r\n");
 
-    // For snapshot requests we will handle things differently after this point
+    if (NewVersion) {
+        client_message_out.append(_("This update is ") + GithubReleaseType + "\r\n\r\n");
+    } else {
+        client_message_out.append(_("The latest release is ") + GithubReleaseType + "\r\n\r\n");
+        client_message_out.append(_("WARNING: You are running a version that is higher than the latest release.") + "\n");
```

Comment:
Doesn't this also get appended if the current version == github version so that there's no new version?

Also a bit of an edge case but the original comparison above would go off in a case where latest github release is something like v5.7.1 and local version is v5.8.0 (only you'd encounter such a case probably) I think.

Reply:
I was making changes to deal with the current version case as you were looking at this. See the force push.